06_gpu_and_ml/llm-serving/sgl_vlm.py

```diff
@@ -50,21 +51,26 @@
 MODEL_CHAT_TEMPLATE = "qwen2-vl"
 
 # We download it from the Hugging Face Hub using the Python function below.
+# We'll store it in a [Modal Volume](https://modal.com/docs/guide/volumes)
+# so that it's not downloaded every time the container starts.
 
+MODEL_VOL_PATH = Path("/models")
+MODEL_VOL = modal.Volume.from_name("sgl-vlm-model", create_if_missing=True)
+volumes = {
+    MODEL_VOL_PATH: MODEL_VOL,
```

Comment:
nit: prefer leaving off trailing commas so that our formatter doesn't create extra lines. vertical space is precious in code rendered inside prose!